src/db/access_keys.rs

```diff
@@ -49,6 +51,29 @@ impl AccessKey {
                             permission: AccessKeyPermission::NotApplicable,
                         }
                     }
+                    near_indexer::near_primitives::views::ActionView::Transfer { .. } => {
+                        if receipt.receiver_id.len() == 64usize {
```

Comment:
To avoid nesting hell, I would invert the condition and `continue` early:

```
                        if receipt.receiver_id.len() != 64 {
                             continue;
                        }
                        ...
```